e2e/tests/components/select/forms/select-forms.e2e-spec.ts

```diff
@@ -589,4 +589,21 @@ describe('Select (forms) Tests', () => {
 
     });
 
-});
\ No newline at end of file
+    it('should correctly validate the tag input values', async () => {
+        await page.clickOnCheckbox(page.checkboxMulti);
+        await page.clickOnDropdown(true);
+
+        await page.clickOnCountry(true, 0);
+        expect(await page.confirmClassExists(page.dropdown, 'ng-invalid')).toBe(true);
+
+        await page.clickOnCountry(true, 1);
+        expect(await page.confirmClassExists(page.dropdown, 'ng-invalid')).toBe(false);
+
+        await page.clickOnCountry(true, 2);
+        await page.clickOnCountry(true, 3);
+        expect(await page.confirmClassExists(page.dropdown, 'ng-invalid')).toBe(true);
+
+        expect(await imageCompare('select-forms-invalid')).toEqual(0);
+    });
```

Comment:
Open source and MF screenshots look different. Feel free to ignore, if it is fine.
![screenshot different](https://user-images.githubusercontent.com/37620152/99642348-6b039500-2a71-11eb-91ab-cf3d937e0afa.png)

Reply:
Yep, this is the correct behaviour. The invalid state of the tag input on UXAspects has a red border the whole way around and in MF it only has a red underline